ntoskrnl/mm/balance.c

```diff
@@ -320,32 +313,6 @@ MiBalancerThread(PVOID Unused)
         {
             ULONG InitialTarget = 0;
 
-#if (_MI_PAGING_LEVELS == 2)
```

Comment:
Does this destroy anything? Or is the whole paging thing fake anyway?

Reply:
This function is never called outside the "balancer thread"